src/main/kotlin/io/sip3/salto/ce/rtcp/RtcpHandler.kt

```diff
@@ -66,13 +68,61 @@ open class RtcpHandler : AbstractVerticle() {
         }
 
         vertx.eventBus().localConsumer<Packet>(RoutesCE.rtcp) { event ->
-            try {
-                val packet = event.body()
-                handleRaw(packet)
-            } catch (e: Exception) {
-                logger.error(e) { "RtcpHandler 'handleRaw()' failed." }
+            val packet = event.body()
+            if (packet.source == "hep3") {
+                try {
+                    handleHep(packet)
+                } catch (e: Exception) {
+                    logger.error(e) { "RtcpHandler 'handleHep()' failed." }
+                }
+            }
+            if (packet.source == "sip3") {
+                try {
+                    handleRaw(packet)
+                } catch (e: Exception) {
+                    logger.error(e) { "RtcpHandler 'handleRaw()' failed." }
+                }
+            }
+        }
+    }
+
+    open fun handleHep(packet: Packet) {
+        val json = JsonObject(packet.payload.toString(Charset.defaultCharset()))
+        try {
+            if (json.getJsonObject("sender_information").getLong("ntp_timestamp_sec").toInt() != 0
```

Comment:
It is enough to check that `sender_information` exists, as far as I see from this code: https://github.com/sipcapture/heplify/blob/master/protos/rtcp.go